damus/Views/BannerImageView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  BannerImageView.swift
+//  damus
+//
+//  Created by Jason Jōb on 2023-01-10.
+//
+
+import SwiftUI
+import Kingfisher
+
+struct InnerBannerImageView: View {
+    let url: URL?
+    let pubkey: String
+
+    var body: some View {
+        ZStack {
+            Color(uiColor: .systemBackground)
+            
+            AsyncImage(url: url) { image in
```

Comment:
We want to use KingFisher here, AsyncImages are not cached

Reply:
Yeah, I wondered about that. I initially tried to use KingFisher but couldn't find a way to make a KFAnimatedImage or KFImage scale correctly like setting the resizable property on a UIImage (or AsyncImage) but I'll take another stab at it. Thanks for the feedback!

Reply:
Got it! Updated to use KFAnimatedImage for the banner image.